tests/contracts/token_fixtures.py

```diff
@@ -27,7 +27,7 @@ def ganachecli_command():
     if "GANACHE_BINARY" in os.environ:
         return os.environ["GANACHE_BINARY"], 5
     else:
-        return 'ganache-cli', 2
+        return 'ganache-cli', 8
```

Comment:
This is probably a remnant of something and needs to be reverted, right?

Reply:
Yes, thanks for pointing it out, reverting it now